app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectsListActivity.kt

```diff
@@ -22,12 +22,20 @@ import com.sdp13epfl2021.projmag.database.Utils
 import com.sdp13epfl2021.projmag.model.ProjectFilter
 
 class ProjectsListActivity : AppCompatActivity() {
+
     private lateinit var itemAdapter: ProjectAdapter
     private lateinit var recyclerView: RecyclerView
-
+    private lateinit var appliedProjects: MutableList<ProjectId>
+    private lateinit var utils: Utils
```

Comment:
If this variable is only used for ProjectAdapter in onCreate, there is no reason to store it in a var ?

Reply:
Are you talking utils?
If so, it is also used in updateAppliedProjects() which is called not only in onCreate but also in onResume()

Reply:
yes, I didn't see it the first time